taskfile/node_http.go

```diff
@@ -16,9 +16,8 @@ import (
 
 // An HTTPNode is a node that reads a Taskfile from a remote location via HTTP.
 type HTTPNode struct {
-	*BaseNode
-	URL        *url.URL // stores url pointing actual remote file. (e.g. with Taskfile.yml)
```

Comment:
question / nitpick : you renamed `URL` to `url` in the `node_git` but not for the `node_http`, should we do the same here ? or this field has to be public ?

Reply:
Honestly just missed it. Thanks!